aat/src/resources/features/F-018/S-014-Verification-call.td.json

```diff
@@ -0,0 +1,42 @@
+{
+  "title": "Get Activity by case Id",
+  "_guid_": "S-014-Verification-call",
+  "productName": "Case Activity API",
+  "operationName": "GET Activity",
+  "method": "GET",
+  "uri": "/activity/cases/{cid}/activity",
+  "specs": [
+    "to get the activity on the case above"
+  ],
+  "users": {
+    "invokingUser": {
+      "username": "[[$CCD_CASEWORKER_AUTOTEST_EMAIL]]",
+      "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+    }
+  },
+  "request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers",
+      "Content-Type": "application/json"
+    },
+    "pathVariables": {
+      "cid": "${[scenarioContext][parentContext][childContexts][Standard_Full_Case_Creation_Data][testData][actualResponse][body][id]}"
+    }
+
+
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body": {
+      "arrayInMap":[
+        {
+          "caseId":"[[ANYTHING_PRESENT]]",
```

Comment:
The id of the case is already available in the broader context. Please check the value in the response here against the value fed to the request that adds the activity. Otherwise the test will be significantly weaker.

Reply:
`"caseId": "${[scenarioContext][testData][request][pathVariables][cid]}"`

Reply:
updated